docs/usage.md

```diff
@@ -235,3 +235,68 @@ and `--project` options are already specified by the tools are should not be inc
 ```bash
 $ evg-module-manager commit-queue
 ```
+
+## Workflows
```

Comment:
This is a mostly notes on what needs to be supported for me to fully dogfood the tool. These still need to be fleshed out more.